packages/kubernetes-1.28/dockershim-symlink.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=/bin/ln -sf /run/containerd/containerd.sock /run/dockershim.sock
```

Comment:
This may not be necessary anymore. I started to look in to it when the last variants were added, but will have to look in my notes to see what the situation was then.

Reply:
The symlink is there to allow container workloads that still assume the container runtime socket to be `dockershim.sock` to continue to work. I'm not opposed to removing this, but maybe we should start with a deprecation notice before complete removal? not sure.